src/UnitTests/Typing/TypedConstantRewriterTests.cs

```diff
@@ -321,8 +363,64 @@ public void Tcr_TypelessPointer()
             c.TypeVariable.DataType = PrimitiveType.SegPtr32;
             c.TypeVariable.OriginalDataType = PrimitiveType.SegPtr32;
 
-            var e = tcr.Rewrite(c, null, false);
+            var e = RewritePointer(c);
             Assert.AreEqual("&seg0C00->t0200", e.ToString());
         }
+
+        [Test(Description = "Don't rewrite pointer to double constant at read-only memory")]
+        public void Tcr_Real64_ReadOnly_Pointer()
+        {
+            Given_TypedConstantRewriter();
+            Given_UInt64(0x4029800000000000, 0x00100000); // 12.75
```

Comment:
Consider renaming this `Given_UInt64_At` to clearly indicate the intent.

Reply:
Done in 344c373